tests/annotation.py

```diff
@@ -0,0 +1,12 @@
+from .testutils import VerminTest, detect
+
+class VerminModuleTests(VerminTest):
+
+  def test_return_annotation(self):
+    self.assertOnlyIn(3.5, detect("def foo() -> str:\n    return ''"))
+
+  def test_arg_annotation(self):
+    self.assertOnlyIn(3.5, detect("def foo(bar: int):\n    pass"))
+
+  def test_var_annotation(self):
+    self.assertOnlyIn(3.6, detect("a: int = 5"))
```

Comment:
In these tests you have to make sure of the minimum version of Python used to interpret it, as you can see in the [travis tests](https://travis-ci.org/netromdk/vermin/jobs/406182650#L579). This is only necessary when the syntax is invalid in earlier python versions.

You can use `testutils.current_version()`